### PR TITLE
A slightly better readme & a mentee task / question list

### PR DESCRIPTION
**Pull request by brettwise**
Completes work outlined in issue #1.

Additionally adds a bunch of questions to help get folks started in their exploration and involvement of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,48 @@
...
+As of Jan 27, 2022 Evo is just a functional core that we pass data through.
 
-What actions am I going to take on the matrix?
+So right now you can do 1 of 3 things: run tests, play with functions already built in `iex`, or help build this thing.
 
-Definitely going to randomly write to it as stuff moves around as well as dies on the map and fertilizes squares.
+  * Install dependencies: `mix deps.get`
+  * Run tests: `mix test`
+  * To play with code already written: `iex -S mix`
+  * Helping build this thing!
```
Comment (thejohncotton) [inline README.md]:
```suggestion
  * Help build this thing!
```

File: README.md [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
```
Comment (thejohncotton) [inline README.md:72]: Might be good to link to a resource that describes the collaborative workflow preference: https://gist.github.com/Chaser324/ce0505fbed06b947d962
or https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow#:~:text=Forking%20is%20a%20git%20clone,org%2FuserA%2Fopen%2Dproject for a more official doc

**Review by thejohncotton — COMMENTED** [review]
Just a couple of quick thoughts for clarity. It's a nice improvement for sure.

File: README.md (hunk 2 above)
Comment (brettwise) [reply README.md:72]: Nice! Thanks for surfacing these. Added that atlassian link and a section to the readme.

**Pull request merged**